contact.html

```diff
@@ -0,0 +1,170 @@
+
+
+    <link href='https://maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css' rel='stylesheet'>
+
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" integrity="sha384-1q8mTJOASx8j1Au+a5WDVnPi2lkFfwwEAa8hDDdjZlpLegxhjVME1fgjWPGmkzs7" crossorigin="anonymous">
+
+    <link  rel="stylesheet" href="css/style.css" type="text/css">
+
+  </head>
+  <body class="contact">
+    <header class="header">
+      <div class="container">
+        <div class="row">
+          <h1 class="logo">
+            <a href="index.html">Kobe</a>
+          </h1>
+          <nav class="main-nav">
+            <div class="clearfix">
+            <div class="input-group input-group-md">
+              <span class="input-group-addon" id="sizing-addon1">
+                <span class="glyphicon glyphicon-search" aria-hidden="true"></span>
+              </span>
+              <input type="text" class="form-control" placeholder="Search" aria-describedby="sizing-addon1">
+            </div>
+            </div>
+              <ul>
+                <li><a href=index.html>HOME</a></li>
+                <li><a href="history.html">HISTORY</a></li>
+                <li><a href="menu.html">MENU</a></li>
+                <li><a href="gallery.html">GALLERY</a></li>
+                <li><a class="active" href="contact.html">CONTACT</a></li>
+              </ul>
+          </nav>
+        </div>
+      </div>
+</header>
+
+    <section class="section01">
+    <div class="row text-center">
+      <h2 class="hero-text">Contact</h2>
+    </div>
+      <img src="images/placeholder-large6.jpg" alt="..." />
+    </section>
+
+    <section class="section02">
+      <div class="container">
+        <div class="row">
+          <div class="col-md-6">
+            <div class="block">
+              <div class="done">
+              <b>Thank you !</b> We have received your message.
+              </div>
+              	<div class="form">
+              	<form method="post" action="processinput.php">
+              	<div class="element">
+              		<label>Name</label>
+              		<input type="text" name="name" class="text" />
+              	</div>
+              	<div class="element">
+              		<label>Email</label>
+              		<input type="text" name="email" class="text" />
+              	</div>
+
+              	</div>
+              	<div class="element">
+              		<label>Message</label>
+              		<textarea name="message" class="text textarea" /></textarea>
+              	</div>
+              	<div class="element">
+
+              		<input type="submit" id="submit"/>
+              		<div class="loading"></div>
+              	</div>
+              	</form>
+              </div>
+          	</div>
+          </div>
+        </div>
+
+
+      <div class="container">
+        <div class="row">
+          <div class="col-md-6">
```

Comment:
The reason why the contact form & calendar are not lining up is because they're both in different `.container` & `.row`. 

Move this `.col-md-6`(line 99) up after line 92 or 93.. Please fix indentation, can't tell where your closing tag for `.container`(line 62), `.row`(line 63), & `.col-md-6`(line 64) ends.